config/schemas/settings.json

```diff
@@ -381,6 +381,14 @@
 				"autoCheckRepositories" : {
 					"type" : "boolean",
 					"default" : true
+				},
+				"updateOnStartup" : {
+					"type" : "boolean",
+					"default" : true
+				},
+				"updateConfigUrl" : {
+					"type" : "string",
+					"default" : "https://raw.githubusercontent.com/Nordsoft91/vcmi-autoupdate/main/autoUpdate.json"
```

Comment:
At list it is possible to fetch the list of updates from downloads.vcmi.eu But the idea of this is to allow downgrade in case of instability of latest release.

Reply:
GitHub api is not an option - it has very limited restrictions for usage without authentication